tests/functional/fx_fennec_v1_sign_up.js

```diff
@@ -58,6 +58,20 @@ define([
         })
 
         .then(FunctionalHelpers.testIsBrowserNotified(self, 'fxaccounts:can_link_account'))
```

Comment:
from irc:
```
<vladikoff>	stomlinson, in choose what to sync, I'm trying to check to make sure the account is NOT created until the user choose what to sync
```
I remember we talked about this and there is an issue with getting into weird states if account is created before the choice made. This should happen after the user submits the  `choose what to Sync page`